todoie/Bindable.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Bindable.swift
+//  todoie
+//
+//  Created by Mustafa Khalil on 1/2/19.
+//  Copyright © 2019 Aurora. All rights reserved.
+//
+
+import Foundation
+
+class Bindable<T>{
+    
+    // oberver is the function that will be called when the value changes with the value
```

Comment:
This is going to be the observer that should invoke the code that we set in the view controller.